README.md

```diff
@@ -1,6 +1,6 @@
 <h1 align="center">
-    <img width="343" src=".github/wagtail.svg#gh-light-mode-only" alt="Wagtail">
-    <img width="343" src=".github/wagtail-inverse.svg#gh-dark-mode-only" alt="Wagtail">
+    <img width="343" src=".github/wagtail(W).svg#gh-light-mode-only" alt="Wagtail">
+    <img width="343" src=".github/wagtail-inverse(B).svg#gh-dark-mode-only" alt="Wagtail">
```

Comment:
Best to replace the existing images instead of adding new ones.

Reply:
Okay, working on it